# Da Física Clássica à Teoria Quântica

Acreditando que átomos eram bolas elásticas, conseguiram explicar vários fenômenos observados. Mas e a estabilidade das moléculas? Como se ligam e continuam assim?
Max Plank por volta de 1900 descobriu que a energia era emitida em pacotes, ou seja, em determinadas quantidades. Uma analogia feita pelo físico Marcelo Gleiser que acho bem legal é a seguinte:\
O motivo de demorarmos tanto a perceber que a energia vem sempre em pacotes é porque vivemos em um mundo de milionários, sem contatos com moedas de 1 centavo. Mas 1 centavo existe e todo o nosso dinheiro é um múltiplo dele.\

## Ondas
Uma onda é uma forma de transmitir energia, vibracionalmente. Ela possui um comprimento, frequência e amplitude.

<img src="onda.png" width=50% >

A sua velocidade pode ser expressa como:

\begin{equation}
v = \lambda f
\end{equation}

Em que $\lambda$ é o comprimento de onda e $f$ a frequência (em Hz).
Podemos também escrever da seguinte forma:

\begin{equation}
f = \frac{1}{T}
\end{equation}

\begin{equation}
v = \frac{\lambda}{T}
\end{equation}

Em que T é o período.

In [ ]:
#v = lam * f
#lam = v/f
#f = v/lam

In [1]:
#Definindo as funções equivalentes (isolando v, lambda ou f):

def velocidade (lam, f):
    return lam * f

def comprimento (v, f):
    return v/f

def frequencia (v, lam):
    return v/lam


Exemplo 1 - O comprimento de luz verde tem comprimento próximo a 522nm. Qual o valor de f?\

Velocidade da luz: 3 x 10⁸ m/s.

In [7]:
v = 3e8
lam = 522e-9 #1 nano metro = 10^-9 m
print('Frequência = ', frequencia (v, lam))

Frequência =  574712643678160.9


Ou seja, passam 574712643678160.9 "ondas" a cada segundo por um mesmo ponto.\

Exemplo 2 - Qual o comprimento de onda de uma onda eletromagnética com $f = 3.64 x 10^7$ Hz?

In [8]:
f = 3.64e7

print('Comprimento = ', comprimento(v, f))

Comprimento =  8.241758241758241


# Teoria Quântica de Planck

Planck chamou de quantun a menor quantidade de energia que pode ser emitida ou absorvida na forma de radiação eletromagnética. Essa energia pode ser expressa como:

\begin{equation}
E = hf
\end{equation}

Ou

\begin{equation}
E = h\frac{c}{\lambda}
\end{equation}

Em que $f$ é a frequência, $h$ a constante de Planck, $c$ a velocidade (da luz) e $\lambda$ o comprimento de onda. Sendo que $h = 6.63 x 10^{-34}$ Jxs.

In [10]:
def Energia_f(f):
    
    '''A frequência deve ser em Hz'''
    
    return 6.63e-34 * f

def Energia_l(lam):
    
    '''O comprimento deve ser em metro'''
    
    return 6.63e-34 * (3e8/lam)

Exemplo 1 - Qual a energia de um fóton com $\lambda = 5.00 x 10^{4}$ nm (infravermelho) e outro com $\lambda = 5.00 x 10^{-2}$ nm (raio X).

In [18]:
#Como foi dado o comprimento, usaremos Energia_l

print('Energia do fóton 1: ',Energia_l(5e-5),'j') #lembrando que foi dado em nm e não em m. Aqui fizemos a 
print('Energia do fóton 2: ',Energia_l(5e-11),'j') #conversão

Energia do fóton 1:  3.978e-21 j
Energia do fóton 2:  3.978e-15 j


Exemplo 2 - A energia de um fóton é de $5.87 x 10^{-20} j$. Qual o seu comprimento de onda?

In [6]:
#Fazendo uma nova função
def Comprimento(E):
    return 6.63e-34*3e8/E

In [25]:
print('Comprimento de onda: ', Comprimento(5.87e-20)/1e-9, 'nm.')
print('Comprimento de onda (em m): ', Comprimento(5.87e-20), 'm')

Comprimento de onda:  3388.4156729131173 nm.
Comprimento de onda (em m):  3.3884156729131173e-06 m


# Efeito Fotoelétrico

Em 1905, Albert Einstein propôs uma solução para o Efeito Fotoelétrico (eletrons são expelidos da superfície de alguns metais quando expostos a uma luz a partir de uma frequência limite).
Ele sugeriu que um feixe de luz seria um feixe de partículas, os fótons.
Os elétrons estão aprisionados em um metal e para arrancá-los é necessário uma certa frequência suficientemente elevada. Se a frequência dos fótons for tal que $hf$ é exatamente igual à energia para arrancá-los, eles serão apenas arrancados. Se $hf$ for maior que essa energia necessária, eles serão arrancados e vão adquirir energia cinética.
Podemos expressar como:

\begin{equation}
hf = K + U
\end{equation}

Em que U é a energia cinética e K a energia para arrancá-lo, lembrando que $hf$ é a energia que vimos acima. Rearranjando:

\begin{equation}
U = hf - K
\end{equation}

Ao escrever a equação dessa forma, podemos entender melhor. Quanto mais energético o fóton (maior for  a sua frequência), maior será a energia do elétron expelido.

O problema disso é que consideramos a luz agora não como uma onda, mas sim como uma partícula! Como essa explicação é condizente com o que se observa nos experimentos, admite-se que a luz tem as propriedas de onda e partícula! É essa a dualidade tão comentada.

In [13]:
#Energia cinética do elétron

def U(f, K):
    ''' f em Hz e K em joule'''
    return 6.63e-34 * f - K

## Teoria de Bohr do Átomo de Hidrogênio

Espectros de emissão são espectros de linhas de radiação emitida pelas substâncias. Para ajudar a explicar isso, Bohr desenvolveu o seu modelo.\
O "átomo de Bohr" incluía elétrons movimentando-se em órbitas circulares, mas só podiam estar em determinadas órbitas.\
As energias relativas ao movimento do elétron nas órbitas permitidas deviam ter valores quantizados. Ele demonstrou que o elétron de um átomo de hidrogênio pode apresentar valores de energias dados por:

\begin{equation}
E_{n} = -R_{H}(\frac{1}{n^{2}})
\end{equation}

$R_{H}$ é a constante de Rydberg, tal que $R_{H}=2.18x10^{-18}J$. $n$ é o número quântico principal. $n = 1$ é o estado de energia mais estável, chamado de estado/nível fundamental. Para $n > 1$, temos os chamados estados excitados. Nesses estados, o elétron é mais instável.
Mas imagine que um elétron tenha saído de um estado $n_{i}$ para um $n_{f}$. Para a diferença de energia, teremos:

\begin{equation}
\Delta E = hf = R_{H}(\frac{1}{n_{i}^{2}} - \frac{1}{n_{f}^{2}})
\end{equation}

Quando o elétron perde energia, então $n_{i} > n_{f}$. Quando o elétron absorve energia, então $n_{i} < n_{f}$.

In [1]:
#Fazendo uma função para a energia de determinado nível:

def Energia_Bohr(n):
    return - 2.18e-18 * (1/(n**2))

def DE_Bohr(ni, nf): #D significa Delta --> variação
    return 2.18e-18 * (   (1/(ni**2))   -  (1/nf**2)  )

Exemplo 1 - Qual o comprimento de onda de um fóton emitido por um elétron que passou do estado $n_{i} = 5$ para $n_{f} = 2$?

In [5]:
#Para resolver, vamos primeiro achar a variação de energia
E = DE_Bohr(5,2)
print('Diferença de energia = ',E,'J')

Diferença de energia =  -4.578e-19 J


Pensando na equação abaixo, precisaremos apenas de isolar o comprimento de onda e substituir os valores:

\begin{equation}
E = h\frac{c}{\lambda}
\end{equation}

In [14]:
import numpy as np
#Agora, vamos obter o comprimento de onda usando a função Comprimento(), que fizemos anteriormente.

#np.abs nos dá o valor absoluto, ou seja, o módulo.
print('Comprimento de onda do elétron: ', Comprimento(np.abs(E)),'m')
print('Comprimento de onda do elétron (em nm): {:.0f} nm'.format(Comprimento(np.abs(E)) * 1e9))

Comprimento de onda do elétron:  4.344692005242464e-07 m
Comprimento de onda do elétron (em nm): 434 nm


Exemplo 2 - Qual o comprimento de onda de um fóton emitido durante uma transição de $n_{i} = 6$ para $n_{f} = 4$?

In [20]:
#Faremos direto agora
print('Comprimento: ', Comprimento(np.abs(DE_Bohr(6,4))),'m')
print('Comprimento (em nm): {:.0f} nm'.format(Comprimento(np.abs(DE_Bohr(6,4))) * 1e9))

Comprimento:  2.627669724770642e-06 m
Comprimento (em nm): 2628 nm


Essas transições dos elétrons do átomo de hidrogênio recebem os nomes de seus descobridores:
<img src="Níveis de energia.jpg" width=50% >

Fonte da imagem: < http://fisicaquimicanet.blogspot.com/2010/01/atomo-de-hidrogenio-e-estrutura-atomica.html >.

Fonte do texto:\
CHANG, Raymond. "Química Geral, conceitos essenciais". 4 ed. Porto Alegre: AMGH, 2010.